examples/rust/jsonpath/Makefile

```diff
@@ -0,0 +1,38 @@
+MODULE := jsonpath
```

Comment:
Thanks for adding this Makefile!  I didn't realize it was missing.

Reply:
My apologies, I completely missed this README.  Updated in cae39a3

Reply:
No worries on the README.  I forgot to tell you about it!